nix/default.nix

```diff
@@ -46,129 +45,131 @@
   enableNvidiaPatches ? false,
   nvidiaPatches ? false,
   hidpiXWayland ? false,
-}:
-assert lib.assertMsg (!nvidiaPatches) "The option `nvidiaPatches` has been removed.";
-assert lib.assertMsg (!enableNvidiaPatches) "The option `enableNvidiaPatches` has been removed.";
-assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been removed. Please refer https://wiki.hyprland.org/Configuring/XWayland";
-  stdenv.mkDerivation {
-    pname = "hyprland${lib.optionalString debug "-debug"}";
-    inherit version;
+}: let
+  adapters = lib.concatLists [
+    [stdenvAdapters.useMoldLinker]
+    (lib.optional debug stdenvAdapters.keepDebugInfo)
```

Comment:
Is this actually needed when dontStrip is set?

Reply:
This also provides proper symbols for use in `gdb`'s `tui` mode for example.